drivers/hetznercloud/flags.go

```diff
@@ -58,4 +57,30 @@ var DriverFlags = []cli.Flag{
 		EnvVars:  []string{"WOODPECKER_HETZNERCLOUD_LABELS"},
 		Category: category,
 	},
+	&cli.StringSliceFlag{
+		Name:     "hetznercloud-firewalls",
+		Usage:    "hetzner cloud firewalls",
```

Comment:
to be consistent it would need to be sth like: `names of hetzner cloud firewalls`